### PR TITLE
DEV-13404 upgrading fabric 8

### PR DESCRIPTION
**Pull request by mayhuberman**

### CODE REVIEW

File: apollo-backend/src/main/java/io/logz/apollo/transformers/deployment/DeploymentLabelsTransformer.java [hunk 1]
```diff
@@ -44,6 +47,16 @@ public Deployment transform(Deployment deployment,
...
+//        List<LabelSelectorRequirement> matchExpressions = deployment.getSpec().getSelector().getMatchExpressions();
+//        LabelSelectorRequirement labelSelectorRequirement = new LabelSelectorRequirementBuilder()
+//                .withNewKey(ApolloToKubernetes.getApolloDeploymentUniqueIdentifierKey())
+//                .withOperator("In")
+//                .addNewValue(ApolloToKubernetes.getApolloDeploymentUniqueIdentifierValue(apolloEnvironment, apolloService, Optional.ofNullable(apolloDeployment.getGroupName())))
+//                .build();
+//
+//        matchExpressions.add(labelSelectorRequirement);
+//        deployment.getSpec().getSelector().setMatchExpressions(matchExpressions);
+
```
Comment (ShaharGotshtat) [inline apollo-backend/src/main/java/io/logz/apollo/transformers/deployment/DeploymentLabelsTransformer.java]: Remove comments

Comment (mayhuberman) [reply apollo-backend/src/main/java/io/logz/apollo/transformers/deployment/DeploymentLabelsTransformer.java:59]: @igalratiner

File: apollo-backend/src/test/java/io/logz/apollo/KubernetesHandlerTest.java [hunk 2]
```diff
@@ -249,7 +259,7 @@ private String getExpectedEnvStatus(Deployment deployment) {
         DeployableVersionDao deployableVersionDao = standaloneApollo.getInstance(DeployableVersionDao.class);
 
         String commitShaFromDeployableVersion = deployableVersionDao.getDeployableVersion(deployment.
-                getDeployableVersionId()).getGitCommitSha();
+                                                                                                            getDeployableVersionId()).getGitCommitSha();
         // String commitShaFromOtherDeployableVersion = deployableVersionDao.getDeployableVersion(finishedDeploymentForEnvTest.getDeployment()
         //         .getDeployableVersionId()).getGitCommitSha();
```
Comment (asafalima) [inline apollo-backend/src/test/java/io/logz/apollo/KubernetesHandlerTest.java:264]: remove

File: apollo-backend/src/test/java/io/logz/apollo/KubernetesHandlerTest.java [hunk 3]
```diff
@@ -270,31 +280,37 @@ private static void setMockDeploymentStatus(RealDeploymentGenerator realDeployme
...
+                                              .withLabels(
+                                                      ImmutableMap.of(ApolloToKubernetes.getApolloCommitShaKey(),
+                                                              realDeploymentGenerator.getDeployableVersion().getGitCommitSha())
+                                              )
+                                              .endMetadata()
+                                              .build())
+                              .build())
+              .always();
+
+        logger.info("");
```
Comment (asafalima) [inline apollo-backend/src/test/java/io/logz/apollo/KubernetesHandlerTest.java:309]: remove

File: apollo-backend/src/test/java/io/logz/apollo/KubernetesHandlerTest.java [hunk 4]
```diff
@@ -321,33 +337,31 @@ private static void setMockPodLogsAndStatus(RealDeploymentGenerator realDeployme
...
+              .andReturn(HttpStatus.OK, log)
+              .always();
+
+        server.expect()
+              .get()
+              .withPath(String.format("/api/v1/namespaces/%s/pods/%s", realDeploymentGenerator.getEnvironment().getKubernetesNamespace(), podStatus.getName()))
+              .andReturn(HttpStatus.OK, pod)
+              .always();
+
+        logger.info("");
```
Comment (asafalima) [inline apollo-backend/src/test/java/io/logz/apollo/KubernetesHandlerTest.java:365]: remove

**Pull request merged**